packages/components/src/utils/config-values.js

```diff
@@ -40,6 +40,7 @@ export default {
 	controlPaddingXLarge: `calc(${ CONTROL_PADDING_X } * 1.3334)`,
 	controlPaddingXSmall: `calc(${ CONTROL_PADDING_X } / 1.3334)`,
 	controlBackgroundColor: COLORS.white,
+	controlBackgroundColorHover: 'rgba(0, 0, 0, 0.5)',
```

Comment:
I took this straight from G2's [controlBackgroundColorHover](https://github.com/ItsJonQ/g2/blob/60816e28df2ea7910997390ba16cd1d2d54c3197/packages/styles/src/theme/config.js#L91).

What's the best way to migrate these styles and know which is the default?

Or are we going to clean up/make things consistent across the board later?